app/assets/javascripts/app/users/user.js

```diff
@@ -5,42 +5,27 @@
     .module('council.users')
     .controller('EditUserCtrl', EditUserController);
 
-  function EditUserController(User, userId, DS) {
+  function EditUserController($state, User, userId) {
     var ctrl = this;
-
-    User.find(userId).then(resetUser);
-
-    ctrl.changePassword = changePassword;
     ctrl.message = '';
-    cleanPasswordFields();
-
-    function changePassword() {
-      if (ctrl.password !== ctrl.passwordConfirmation) return;
-
-      ctrl.user.update({
-        password: ctrl.password,
-        password_confirmation: ctrl.passwordConfirmation,
-        username: ctrl.username })
-        .then(passwordUpdated, passwordNotUpdated);
+    ctrl.data = {};
+    ctrl.userUpdated = userUpdated;
+    ctrl.userNotUpdated = userNotUpdated;
 
-      cleanPasswordFields();
-    }
+    User.find(userId).then(resetUser);
 
-    function resetUser(user) {
-      ctrl.user = user;
+    function resetUser() {
+      ctrl.user = User.get(userId);
+      ctrl.data.username = ctrl.user.display_name;
+      ctrl.data.color = ctrl.user.color;
     }
 
-    function passwordUpdated() {
-      ctrl.message = 'User updated with success';
+    function userUpdated() {
+      $state.go('discussions.index');
     }
 
-    function passwordNotUpdated() {
+    function userNotUpdated() {
       ctrl.message = 'Failed to update';
```

Comment:
I don't think the outer controller should know about this property. Perhaps you don't even need to define this callback at all, and let let form's controller deal with this on its own